pom.xml

```diff
@@ -174,7 +175,12 @@
             <groupId>org.dataconservancy.pkgtool</groupId>
             <artifactId>dcs-packaging-tool-model-builder</artifactId>
         </dependency>
-    </dependencies>
+        <dependency>
+            <groupId>org.dataconservancy.packaging.shared</groupId>
+            <artifactId>dcs-packaging-shared</artifactId>
+            <scope>compile</scope>
+            <version>1.0.3</version>
```

Comment:
Version doesn't need to be here as it is specified in the `<dependencyManagement>` section.